lib/xlsx_reader/package_loader.ex

```diff
@@ -33,15 +33,16 @@ defmodule XlsxReader.PackageLoader do
   def open(zip_handle, options \\ []) do
     with :ok <- check_contents(zip_handle),
          {:ok, workbook} <- load_workbook_xml(zip_handle, options),
-         {:ok, workbook_rels} <- load_workbook_xml_rels(zip_handle) do
-      package =
-        %XlsxReader.Package{
-          zip_handle: zip_handle,
-          workbook: %{workbook | rels: workbook_rels}
-        }
-        |> load_shared_strings
```

Comment:
in this case when there's an error it seems we'd be crashing today, as `{:error, _}` is not cascading up, fixing this here